### ISSUE TITLE
Unable to post in Read Only channels from Android 

### ISSUE THREAD

**Issue by lbblitz54** [RocketChat/Rocket.Chat.Android#2449]
## Describe the bug

Unable to post in Read Only channels from Android even with Admin Owner or Moderator roles.
## To Reproduce

1. Create/Join read only channel
2. Ensure proper role(s) are assigned 
3. "This room is read only" still apears and unable to post to channel. 

## Expected behavior

User should be able to post to the channel with any of the 3 roles Admin Owner or Moderator
![readonlybug](https://user-images.githubusercontent.com/17748076/62748084-e415df00-ba1c-11e9-81c8-d0cd6035ef2b.png)
![readonlybug2](https://user-images.githubusercontent.com/17748076/62748143-1cb5b880-ba1d-11e9-9422-27948cc3ef97.png)

## Devices and Versions

Your Rocket.Chat.Android version: (e.g. 3.5.0(2074))
Your Rocket.Chat Server version: (e.g. 1.2.0)

Mobile device model and OS version: (e.g. "Nexus 7 - Android 6.0.1")
Samsung Galexy S9 - Android 9.0
Sonim XP8800 - Android 7.1

**Comment by lbblitz54** [comment]
also found that there is only the option to "REPLY" in the android app 
this acts similar to the other platforms options to "quote" vs what i would like which is the typical "reply in thread" which is not an option

**Comment by llacroix** [comment]
you do understand that readonly stands for "no write possible", or in other words... nothing can be added/removed. Seems to work as it should.

**Comment by lbblitz54** [comment]
> you do understand that readonly stands for "no write possible", or in other words... nothing can be added/removed. Seems to work as it should.

Yes @llacroix  i am very well aware of the implications of the "read only" channel and can appreciate your POV as well as the "dummy check" of my experience. 

However, what i am referring to is the ability for the OWNER/ADMIN/MODERATOR of said "read only" channels to be able to write to this channel via the Android App. 

To be crystal clear lets dumb it down a bit: 
  1. I as OWNER of ReadOnly channel can publish to said channel via the Desktop App (of course bc i am owner and someone is suppose to be able to)

 2. I as OWNER of ReadOnly channel can publish to said channel via the iOS App ( of course, why would't I be able to I am the owner)

 3. I as OWNER of ReadOnly channel can publish to said channel via the Web Browser ( you should be getting the trend by now)

 4. I as OWNER of ReadOnly channel CAN NOT publish to said channel via the Android App ( because it is broken... not because i should not be able to write to this channel)

**Comment by DavidGreybosh** [comment]
Find any fix for this issue? Still having the same issue with my version of the app as well.
